pilot/pkg/xds/monitoring.go

```diff
@@ -151,11 +151,21 @@ var (
 		"Total number of updates received by pilot.",
 	)
 
+	pilotSDSSecretErrors = monitoring.NewSum(
+		"pilot_sds_secret_errors_total",
```

Comment:
seems redundant

Reply:
this should get incremented even when there is `no secret found` -- the other two - `CertificateErrors` and `GenericErrors` will be incremented when there is an issue with extracting the key, val